src/spi.rs

```diff
@@ -135,6 +164,27 @@ remap!(Spi3NoRemap, pac::SPI3, false, PB3, PB4, PB5);
 #[cfg(feature = "connectivity")]
 remap!(Spi3Remap, pac::SPI3, true, PC10, PC11, PC12);
 
+macro_rules! remap_slave {
+    ($name:ident, $SPIX:ty, $state:literal, $SCK:ident, $MISO:ident, $MOSI:ident) => {
+        pub struct $name;
+        impl Remap for $name {
+            type Periph = $SPIX;
+            const REMAP: bool = $state;
+        }
+        impl<MODE> Ssck<$name> for gpio::$SCK<Input<MODE>> {}
+        impl<MODE> So<$name> for gpio::$MISO<Alternate<MODE>> {}
+        impl<MODE> Si<$name> for gpio::$MOSI<Input<MODE>> {}
+    };
+}
+
+remap_slave!(Spi1NoRemapSlave, pac::SPI1, false, PA5, PA6, PA7);
+remap_slave!(Spi1RemapSlave, pac::SPI1, true, PB3, PB4, PB5);
+remap_slave!(Spi2NoRemapSlave, pac::SPI2, false, PB13, PB14, PB15);
+#[cfg(any(feature = "high", feature = "connectivity"))]
+remap_slave!(Spi3NoRemapSlave, pac::SPI3, false, PB3, PB4, PB5);
+#[cfg(feature = "connectivity")]
+remap_slave!(Spi3RemapSlave, pac::SPI3, true, PC10, PC11, PC12);
+
```

Comment:
What sense of duplicating?

Reply:
https://github.com/stm32-rs/stm32f1xx-hal/commit/bb248dff6ce4271feb7248f006a341b3490e0c8a

Reply:
Don't see any changes in PR

Reply:
Oh， Still need PR

Reply:
just merge or cherry-pick commit in this branch if all works as expected